src/__tests__/operate.test.js

```diff
@@ -0,0 +1,16 @@
+import operate from '../logic/operate';
+
+describe('Testing calculation operations', () => {
+  test('Should add two numbers', () => {
+    const cal = operate('2', '5', '+');
+    expect(cal).toEqual('7');
+  });
+  test('Should subtract two numbers', () => {
+    const cal = operate('2', '5', '-');
+    expect(cal).toEqual('-3');
+  });
+  test('Should multiply two numbers', () => {
+    const cal = operate('2', '5', 'x');
+    expect(cal).toEqual('10');
+  });
+});
```

Comment:
- [ ] Well done for testing some scenarios of the `operate.js` module. Please write tests for the "division" and "modulo" operators as well so that all the functionalities of the `operate.js` can be covered by Jest.